src/main/java/com/gooddata/md/visualization/VisualizationAttribute.java

```diff
@@ -30,7 +40,7 @@
      * @param alias alias of attribute
      */
     @JsonCreator
-    public VisualizationAttribute(@JsonProperty("displayForm") final ObjQualifier displayForm,
+    public VisualizationAttribute(@JsonProperty("displayForm") final UriObjQualifier displayForm,
```

Comment:
So this attrobute can never carry displayForm as identifier?

Reply:
Yes, it cannot. The MD server accepts `UriObjQualifier` only. I know that this is an API change, but IMO we (GD) are the only ones who uses this objects for now. And it doesn't make sense to accept identifier qualifier if it cannot be accepted later during persisting object to MD server.